docs/how-to-deploy-on-AWS.md

```diff
@@ -0,0 +1,109 @@
+---
+id: aws
+title: "Amazon Web Services"
+---
+
+This document describes simple steps to setup Verdaccio private registry on Amazon Web Services platform using EC2 service. Considering you already created EC2 Amazon Linux instance, if not then please check this tutorial [AWS EC2 Setup](https://tinyurl.com/y2ozksyc).
```

Comment:
I'd rahter use your original URL, `https://tinyurl.com` might be unvalid eventually.

Reply:
sure

Reply:
Let me update original URL

Reply:
Done. Please review again and approve.